src/app/modules/importanceTransfer/multisig/importanceTransfer.controller.js

```diff
@@ -98,10 +99,16 @@ class MultisigImportanceTransferCtrl {
             this.nodes = this._Nodes.get();
         } else {
             this.nodes = [];
-            // Get supernodes
-            nem.com.requests.supernodes.all().then((data) => {
+            // Get superNodes with pageNumber 0
```

Comment:
Why just the first page? We can take all of them and get random X elements. I would say X = 50 for example (I will have a look if it looks good with that amount of elements).

Reply:
I forgot the apply recursively fetch on this. I will refactor this.